lib/secure_headers.rb

```diff
@@ -141,7 +145,6 @@ def opt_out_of_all_protection(request)
     def header_hash_for(request)
       prevent_dup = true
       config = config_for(request, prevent_dup)
-      config.validate_config!
```

Comment:
This nearly doubles the amount of time it takes to generate a header. I'm not sure it's necessary, really.